Initial Changes for SLES15

Gaurav-kr4 opened the pull request:
1. Building smt packages on SLES15SP4
2. Using firewalld instead of SuSEFirewall
3. Fixed the File permission issue for smt.conf

Gaurav-kr4 committed:
Initial Changes for SLES15

LokeshBabu-K reviewed (commented):
- Commented lines with # can be removed. They were left for initial testing and reversal if required
- Line 58 "($self->setopt(CURLOPT_SSLVERSION, 1 ); # 1 should be CURL_SSLVERSION_TLSv1" in the file /usr/lib/perl5/vendor_perl/5.26.1/SMT/Curl.pm should be changed to support TLSv1.2 and above

Gaurav-kr4 committed:
Removed the commented code

Gaurav-kr4 commented:
> * Line 58 "($self->setopt(CURLOPT_SSLVERSION, 1 ); # 1 should be CURL_SSLVERSION_TLSv1" in the file /usr/lib/perl5/vendor_perl/5.26.1/SMT/Curl.pm should be changed to support TLSv1.2 and above

Hi Lokesh, we will get this fixed in the next set of change

Gaurav-kr4 committed:
Adding popup message for missing certificate

Gaurav-kr4 commented:
Changed the tls version CURLOPT_SSLVERSION to CURL_SSLVERSION_TLSv1_2 in the Curl Perl module
Removed the execution of yast2 ca-mgm and showing popup for the missing certificate
